libsodium-sys/Cargo.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 authors = ["dnaq"]
 build = "build.rs"
-description = "FFI binding to libsodium"
+description = "FFI binding to libsodium generated with bindgen"
```

Comment:
I don't think we need to specify that its bindgen generated in the description

Reply:
Yup.